src/test/java/bio/terra/service/snapshot/SnapshotDaoTest.java

```diff
@@ -276,8 +279,24 @@ public void snapshotEnumerateTest() throws Exception {
         }
 
         MetadataEnumeration<SnapshotSummary> emptyEnum = snapshotDao.retrieveSnapshots(0, 6, null,
-            null, "__", snapshotIdList);
+            null, "__", datasetIds, snapshotIdList);
         assertThat("underscores don't act as wildcards", emptyEnum.getItems().size(), equalTo(0));
+
+        MetadataEnumeration<SnapshotSummary> summaryEnum0 = snapshotDao.retrieveSnapshots(0, 10, null,
```

Comment:
should we try one of these where it's null just in case?

Reply:
we know it will NPE since it directly talks to the Dao---there was a !null check previously, but it was removed since it seemed like overkill. Which is better?